### DISCUSSION TITLE
How to res the username and pass

### DISCUSSION THREAD

**Discussion by MahdiElahi** [dotnet/aspnetcore#55363] [Q&A]
Hi , 
I have web site with 10 users 
i give them to any user one password and username that login to my web site

but for some reason i want this username and password just worked in him devices it's means if some user give his username & pass toh  any one to login to web site , there no access to login form other device

i have 10 user with 10 username and password 
i dont want other pc or other device can login to site execpt these 10 pc 

first i store Ip system and mac address but these is not useful beacause these are can changed

how to do it in blazor .net 8 with identity Authoriztion

**Comment by mkArtakMSFT** [comment]
Hi. It looks like this is a question about how to use ASP.NET Core. While we do our best to look through all the issues filed here, we are not a general-purpose forum. To get a faster response we suggest posting your questions to [StackOverflow](https://www.stackoverflow.com) using the **asp.net-core** tag.

**Comment by RutulPatel8** [comment]
Hi @MahdiElahi,
These questions are not related to .net. It is all about the process which you need to build. I suggest use FCM token but i think it isn't a mobile application.

The big question is how closely you need to stick to your requirement.

There are lots of solutions such as simply authenticating the user, using authentication devices (such as RSA keyfobs), client side certificates - but these methods can be transferred between machines. Browser fingerprinting makes it more difficult, and something like evercookie makes circumvention more difficult (but more likely to fail to work in the first place).

Really we need to know more about whether it's the device or the user you need to authenticate and how strong the requirement to disallow duplicated access is.
